packages/terra-open-source-scripts/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@cerner/terra-open-source-scripts",
+  "private": true,
+  "version": "0.0.0",
+  "description": "Terra package with various scripts for open source development",
+  "main": "index.js",
+  "bin": {
+    "terra": "./bin/terra"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-toolkit.git",
+    "directory": "packages/terra-open-source-scripts"
+  },
+  "keywords": [
+    "terra-open-source-scripts",
+    "Cerner",
+    "Terra"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-toolkit/issues"
+  },
+  "engines": {
+    "node": "^10.13.0"
+  },
+  "homepage": "https://github.com/cerner/terra-toolkit",
+  "scripts": {
+    "compile": "babel src --out-dir lib --copy-files --ignore src/terra-cli",
```

Comment:
Same question as before, won't this need to be compiled to lib once this is released and consumed else where?

Reply:
--copy-files and --ignore the way that is set up will copy the files over but not try to compile the ones that are ignored.

Reply:
For posterity, the way you would both ignore *and* not copy a js file is by using the --no-copy-ignored option.